### PR TITLE
Stepper

### PR DESCRIPTION
**Pull request by sirgawain0x**

### CODE REVIEW

**Comment by sirgawain0x** [comment]
@greenealexander would you mind taking a look at this sometime. Can't figure out how to get content inside each step.

**Comment by greenealexander** [comment]
@gawainb I pushed some commits that hopefully point you in the right direction

https://user-images.githubusercontent.com/17241472/205469726-ed12beb0-e2b8-4674-9c18-7b82f355507a.mov

File: src/components/stepper/StepsComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    },
+  },
+  {
+    label: 'Decision Deadline',
+    props: {
+      time: '12/4/2022',
+      tooltip: 'lorem ipsum',
+    },
+  },
+]
```
Comment (greenealexander) [inline src/components/stepper/StepsComponent.tsx:29]: Added more metadata that can be used when we iterate through them

File: src/components/stepper/StepsComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  const bg = useColorModeValue('#F0F0F0', 'gray.900')
+
+  return (
+    <Flex flexDir="column" width="100%">
+      <Steps checkIcon={FiCheckCircle} activeStep={activeStep} labelOrientation={"horizontal"}>
+        {steps.map(({ label, props }, index) => (
+          <Step label={label} key={label}>
+            <Flex my={4} p={4} bg={bg} borderRadius={10}>
+              <CountDown title="Deadline" {...props} />
+            </Flex>
```
Comment (greenealexander) [inline src/components/stepper/StepsComponent.tsx:44]: Pass the props which contain the metadata added above

**Review by greenealexander — COMMENTED** [review]
@gawainb I pushed some commits that hopefully point you in the right direction.

**Comment by greenealexander** [comment]
@gawainb are you trying to render a different form for each step?

**Pull request merged**